Lib/test/test_asyncio/test_streams.py

```diff
@@ -1214,9 +1214,9 @@ async def main():
                 # can't use assertRaises because that clears frames
                 exc = excs.exceptions[0]
             self.assertIsNotNone(exc)
-            self.assertListEqual(gc.get_referrers(exc), [])
-
-        asyncio.run(main())
+            self.assertListEqual(gc.get_referrers(exc), [main_coro])
```

Comment:
Why this change?

Reply:
Because we are spilling the stack pointer, so the GC can see the locals of the generator.